src/designer/designer_packages/elsa-workflows-designer/src/modules/workflow-definitions/plugins/workflow-definitions-plugin.tsx

```diff
@@ -270,4 +273,30 @@ export class WorkflowDefinitionsPlugin implements Plugin {
   private onImportClicked = async (e: CustomEvent) => {
     await this.import();
   }
-}
+
+  private updateCompositeActivityReferences = async (definition: WorkflowDefinition) => {
+    const notification = NotificationService.createNotification({
+      title: 'Consuming Workflows',
+      id: uuid(),
+      text: 'Consuming workflows are being updated.',
+      type: NotificationDisplayType.InProgress
+    });
+
+    await this.api.updateWorkflowReferences({definitionId: definition.definitionId})
+      .then(async (response) => {
+        debugger;
+        NotificationService.updateNotification(notification, {
+          title: 'Consuming Workflows', 
+          text: 'The following consuming workflows have been successfully updated:\n\n' + response.affectedWorkflows.join('\n'),
```

Comment:
If there were 0 affected workflows, the text should be a little bit different, e.g. `There were no consuming workflows`. Or perhaps the notification shouldn't display in the first place. What do you think?